sbt_common/elasticsearch/src/main/scala/uk/ac/wellcome/elasticsearch/ElasticsearchIndex.scala

```diff
@@ -17,9 +17,12 @@ trait ElasticsearchIndex extends Logging {
 
   def create(indexName: String): Future[Unit] =
     httpClient
-      .execute(createIndex(indexName).mappings {
-        mappingDefinition
-      })
+      .execute(
+        createIndex(indexName)
+          .mappings {
+            mappingDefinition
+          }
+          .shards(1))
```

Comment:
Comment to explain why we use a single shard please. Otherwise we'll forget, and somebody will remove this line in a year's time.